content/chapters/02_supervised_regression/02-03-l1-loss.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: "Chapter 02.02: Linear Regression Models"
-weight: 2002
+title: "Chapter 02.03: Linear Models with L1 Loss"
+weight: 2003
 quizdown: true
 ---
-In this section, we explain how the linear regression model can be used from a machine learning perspective to predict a continuous numerical target variable. We use the concepts of loss function and empirical risk minimization to find the linear model that best fits the data.
+In this section, we introduce L1 loss and elaborate its differences to L2 loss. In addition, we explain how the choice of loss affects optimization & robustness.
```

Comment:
Das `&` Zeichen rendert in der Kapitelübersicht komischerweise als `&amp;`, im Kapitel selbst aber als `&` => weißt Du wieso? Vlt. ist auch nur die Vorschau komisch und auf der "echten" Seite passt es dann? => Mach sonst vielleicht einfach ein `and` draus.